CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ endif()
 install(FILES LICENSE CHANGELOG.md DESTINATION . COMPONENT devel)
 
 if(APPLE)
-    set(CPACK_GENERATOR "TXZ")
+    set(CPACK_GENERATOR "TGZ")
```

Comment:
Nice! This will also speed up CI builds since the slaves were struggling with that big .xz file